src/Schema/Comparator.php

```diff
@@ -48,16 +52,37 @@ public function __construct(?AbstractPlatform $platform = null)
      * Returns a SchemaDiff object containing the differences between the schemas $fromSchema and $toSchema.
      *
      * This method should be called non-statically since it will be declared as non-static in the next major release.
+     * Its third argument should be itself.
      *
      * @return SchemaDiff
      *
      * @throws SchemaException
      */
     public static function compareSchemas(
         Schema $fromSchema,
-        Schema $toSchema
+        Schema $toSchema /*, self $comparator = null*/
```

Comment:
Would be nice to add an `@param` don't you think?

Reply:
Symfony's DebugClassLoader will trigger a deprecation if you add one. Also, about the current logic triggering a deprecation, you might want to have a look at how Symfony does it. I'm on a phone sorry can't link. Maybe @derrabus can help?